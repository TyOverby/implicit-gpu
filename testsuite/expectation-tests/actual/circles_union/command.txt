Serially(
    [
        Define(
            0,
            BasicShape(
                Union(
                    [
                        Terminal(
                            Circle(
                                Circle {
                                    x: 11.0,
                                    y: 11.0,
                                    r: 10.0,
                                    mat: [
                                        1.0,
                                        0.0,
                                        0.0,
                                        1.0,
                                        2.0,
                                        2.0
                                    ]
                                }
                            )
                        ),
                        Terminal(
                            Circle(
                                Circle {
                                    x: 21.0,
                                    y: 21.0,
                                    r: 10.0,
                                    mat: [
                                        1.0,
                                        0.0,
                                        0.0,
                                        1.0,
                                        2.0,
                                        2.0
                                    ]
                                }
                            )
                        )
                    ]
                )
            )
        ),
        Export(
            0
        )
    ]
)