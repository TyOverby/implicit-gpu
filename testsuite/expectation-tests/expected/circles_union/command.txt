Serially(
    [
        Define(
            0,
            BasicShape(
                Union(
                    [
                        Terminal(
                            Circle(
                                Circle {
                                    x: 11.0,
                                    y: 11.0,
                                    r: 10.0,
                                    mat: [I]
                                }
                            )
                        ),
                        Terminal(
                            Circle(
                                Circle {
                                    x: 21.0,
                                    y: 21.0,
                                    r: 10.0,
                                    mat: [I]
                                }
                            )
                        )
                    ]
                )
            )
        ),
        Export(
            0
        )
    ]
)